bakerydemo/base/fixtures/bakerydemo.json

```diff
@@ -2101,7 +2128,7 @@
     "fields": {
       "path": "00010002",
       "depth": 2,
-      "numchild": 6,
+      "numchild": 7,
```

Comment:
The RecipeIndexPage sits under the homepage, and I’ve reordered those child pages so it’s in a more natural place in the site’s main nav.